### ISSUE TITLE
[BUG][ALL VERSIONS]: OWL error when a page is Google translated and OWL modifies a list of elements

### ISSUE THREAD

**Issue by kebeclibre** [odoo/owl#1631]
In odoo:
- have a x2m editable list
- google translate the page in chrome (it seems to replace the targeted text with <font /> elements)
- click on a cell to edit its content 
- CRASH in VLIST

I don't think it is easily fixable, but it had to be noted.

For the record: it odoo's task 4063314

**Comment by ged-odoo** [comment]
yeah, that is not really fixable. Owl (and some (all?) other vdom-based framework as well!) uses empty text nodes as anchor to keep track of various dom locations. It is (currently) not supported to modify the dom of a owl-managed html element.  However, maybe we can think of improvements. Maybe a nicer error message.  Or some kind of recovery mode: if owl detects that the patching operation crashed, it could silently replace the (modified) content by the template content (so, giving back the control to owl). Not sure about this

**Comment by ged-odoo** [comment]
not really fixable
